diffyscan/diffyscan.py

```diff
@@ -149,28 +217,90 @@ def process_config(path: str, recursive_parsing: bool, unify_formatting: bool):
     logger.info(f"Loading config {path}...")
     config = load_config(path)
 
-    github_api_token = load_env("GITHUB_API_TOKEN", masked=True)
     explorer_token = None
     if "explorer_token_env_var" in config:
-        explorer_token = load_env(config["explorer_token_env_var"], masked=True, required=False)
+        explorer_token = load_env(
+            config["explorer_token_env_var"], masked=True, required=False
```

Comment:
All new config values should be properly typed and added to `utils→types→Config`

Reply:
I don't understand what role it plays. If I comment it out or if I add incorrect types, nothing breaks. 
So I suggest to remove it completely

Reply:
done